packages/source/types/data.ts

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable */
```

Comment:
This is temporal because @luisherranz has already created the TSDocs for this file in https://github.com/frontity/frontity/pull/564